services/user.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+from db.models import User

```

Comment:
You imported the `User` model directly from `db.models` which is not recommended. It's better to always use `get_user_model()` to retrieve the user model. This is a code style issue.